### PR TITLE
Add e2e tests for low count alerts

### PR DESCRIPTION
**Pull request by machadovilaca**
**What this PR does / why we need it**:

Add missing e2e tests for components low count  

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by enp0s3 — COMMENTED** [review]
/approve
@machadovilaca Thank you!

note-to-self - low count metrics use `sum` and in that case sum up of `stale` its like `0` therefore its OK to scale to zero. Also because low count considered less than 2, a hard-coded number.

**Review by fossedihelm — COMMENTED** [review]
Thanks, just one typo

File: tests/framework/checks/skips.go [hunk 1]
```diff
@@ -214,12 +214,16 @@ func SkipIfOpenShift4(message string) {
 	}
 }
 
-func SkipIfMigrationIsNotPossible() {
```
Comment (xpivarc) [inline tests/framework/checks/skips.go:217]: Please refrain from introducing new skips or using the existing ones. Rather create a general label in `tests/decorators` that can be used to skip a test based on the test suite user`s needs and knowledge.

Comment (machadovilaca) [reply tests/framework/checks/skips.go]: updated, I don't know if it makes sense this way, please take a look

File: tests/decorators/decorators.go [hunk 2]
```diff
@@ -39,4 +39,5 @@ var (
 	Istio                     = []interface{}{Label("Istio")}
 	InPlaceHotplugNICs        = []interface{}{Label("in-place-hotplug-NICs")}
 	MigrationBasedHotplugNICs = []interface{}{Label("migration-based-hotplug-NICs")}
+	LowCountAlerts            = []interface{}{Label("low-count-alerts")}
```
Comment (xpivarc) [inline tests/decorators/decorators.go:42]: Maybe "single-node" would be more generic?

Comment (fossedihelm) [reply tests/decorators/decorators.go:42]: If I can chime in, labels can be descriptive: `RequiresTwoSchedulableNodes`

Comment (xpivarc) [reply tests/decorators/decorators.go:42]: +1

Comment (machadovilaca) [reply tests/decorators/decorators.go]: I like it. Updated

**Comment by fossedihelm** [comment]
Thank you!
/lgtm
@xpivarc do you want to have another look?

**Pull request merged**